examples/dc/interface/main.tf

```diff
@@ -4,4 +4,5 @@ resource "vkcs_dc_interface" "dc_interface" {
   dc_router_id = vkcs_dc_router.dc_router.id
   network_id = vkcs_networking_network.app.id
   subnet_id = vkcs_networking_subnet.app.id
+  ip_address = "192.168.199.42"
```

Comment:
lets do not add this - many tf users think they must specify all arguments from doc example

Reply:
removed